community/kernel/src/main/java/org/neo4j/kernel/internal/GraphDatabaseAPI.java

```diff
@@ -48,6 +52,43 @@
      */
     URL validateURLAccess( URL url ) throws URLAccessValidationError;
 
-    @Deprecated
```

Comment:
Why this deprecation was removed? Imo it should not be removed but instead method should return `java.io.File` or better `java.nio.file.Path`.